activerecord/lib/active_record/core.rb

```diff
@@ -701,16 +702,16 @@ def strict_loading!(value = true, mode: :all)
 
     attr_reader :strict_loading_mode
 
-    # Returns +true+ if the record uses strict_loading with +:n_plus_one_only+ mode enabled.
-    def strict_loading_n_plus_one_only?
-      @strict_loading_mode == :n_plus_one_only
-    end
-
     # Returns +true+ if the record uses strict_loading with +:all+ mode enabled.
     def strict_loading_all?
       @strict_loading_mode == :all
     end
 
+    # Returns +true+ if the record uses strict_loading with +:n_plus_one_only+ mode enabled.
```

Comment:
Why was this method moved?

It looks unchanged, so if you could rebase this out for a cleaner diff :pray:

Reply:
In the ~2.5 months I've been pushing this PR along, someone else [added the exact same method](https://github.com/rails/rails/commit/715276071f8ab72292b88df5d8d25855a35d4339). I've not been tremendously great at rebasing, so every time I have to fix a conflict I rush through it in the vain hope that I can catch a merge. But, I have fixed it.